app/src/main/java/com/eveningoutpost/dexdrip/Services/DexCollectionService.java

```diff
@@ -140,34 +160,70 @@ public void onSharedPreferenceChanged(SharedPreferences prefs, String key) {
         }
     };
 
+    public static byte getBridgeBattery() {
```

Comment:
I feel like this is a strange way to get a value from the service, I would rather it be stored somewhere and then retrieved by the other class (either in the database or sharedprefs)